.coveralls.yml

```diff
@@ -1 +1 @@
-repo_token: pwvIqtiIaOZI17kbFwZRr0XllixWAWp5u
+repo_token: Tw6HTcXBSQGUQiMcPMMw4xN2vBNqjml8j
```

Comment:
I get the sneaking suspicion that this is set up wrong / this key shouldn't get checked in as plaintext. @jenncoop could you investigate?